packages/core/apps/core-apps-server-internal/src/core_app.ts

```diff
@@ -144,6 +146,36 @@ export class CoreAppsService {
         }
       }
     );
+
+    router.versioned
+      .put({
+        path: '/_settings',
```

Comment:
Using `PUT /_settings` to update the dynamic options. I went with this approach instead of UI Settings because we don't need persistence through restarts... Also, this allows us to set specific settings for each Kibana node in case we have multiple instances.

But happy to iterate on that and move this to UI Settings if we don't want to open this pandora box.